logging/unit_tests/test_client.py

```diff
@@ -571,6 +571,59 @@ def test_list_metrics_with_paging(self):
             },
         })
 
+    def test_get_default_handler(self):
+        import os
+        from google.cloud._testing import _Monkey
+        from google.cloud.logging.handlers.GAEHandler import GAEHandler
+        from google.cloud.logging.handlers.GKEHandler import GKEHandler
+        from google.cloud.logging.handlers.handlers import CloudLoggingHandler
+
+        client = self._makeOne(
+            project=self.PROJECT, credentials=_Credentials(),
+            use_gax=False)
+        client.get_default_handler()
+
+        client = self._makeOne(self.PROJECT, credentials=_Credentials())
+
+        with _Monkey(os, getenv=lambda var: var == 'GAE_APPENGINE_HOSTNAME'):
```

Comment:
1. I am pretty anti-`lambda`
1. Don't do this, just patch `os.environ`.
1. Once you make the env. var. name a module-level constant, refer to that here.